src/commands/init.ts

```diff
@@ -44,13 +49,23 @@ export const builder = (yargs: Argv) =>
       hcOnly: { type: "boolean" },
       dataPath: { 
         type: 'string', 
-        describe: 'The relative path for storing ad4m data', 
+        describe: 'Name of directory to store ad4m data in within the systems app data path', 
         alias: 'rp'
       },
+      networkBootstrapSeed: {
+        type: 'string',
+        describe: 'Path to the seed file',
+        alias: 'nbf'
+      },
+      override: {
+        type: 'boolean',
+        describe: 'Override the existing config file',
+        alias: 'o'
```

Comment:
Here o as alias to too general, maybe don't give it an alias, also overrideSeed/overrideConfig sounds better.